scoring/build_score_inventory.R

```diff
@@ -10,6 +10,7 @@ inventory_df <- arrow::open_dataset(s3) |>
   mutate(reference_date = lubridate::as_date(reference_datetime),
          date = lubridate::as_date(datetime),
          pub_date = lubridate::as_date(pub_datetime)) |>
+  filter(project_id == config$project_id) |>
```

Comment:
FYI @rqthomas I think you'll need to add this to `neon4cast-ci` as well to filter out other forecast challenges.

Reply:
It is better to use this code 
```
s3 <- arrow::s3_bucket(paste0(config$scores_bucket, "/parquet/project_id=",config$project_id), endpoint_override = config$endpoint, anonymous = TRUE)

bucket <- config$scores_bucket
inventory_df <- arrow::open_dataset(s3) |>
  mutate(reference_date = lubridate::as_date(reference_datetime),
         date = lubridate::as_date(datetime),
         pub_date = lubridate::as_date(pub_datetime),
         project_id = config$project_id) |>
  distinct(duration, model_id, site_id, reference_date, variable, date, project_id, pub_date) |>
  collect() |>
  mutate(path = glue::glue("{bucket}/parquet/project_id={project_id}/duration={duration}/variable={variable}"),
         path_full = glue::glue("{bucket}/parquet/project_id={project_id}/duration={duration}/variable={variable}/model_id={model_id}/date={date}/part-0.parquet"),
         endpoint =config$endpoint)
```

because it will mount the scores bucket at the project_id and not have to look at the files from other project_ids

Reply:
oh thanks, will do